CSE115A/rideshare-project
# Create Landing Page Task #6

Problem
=======
As a developer, I want to know if my components are working properly.

Solution
========
Create unit tests to check if functionality of webpage works properly.

Change summary:
---------------
added files `globalSetup.js globalTeardown.js jest.config.js index.test.js`

Steps to Verify:
----------------
First run `yarn` to install all the necessary dependencies. 
Then by running the command `yarn test` allows you to run the tests.
It will run the tests that will check if certain components work properly.
*Tests still need to be implemented: check price
